extensions/omni_schema/src/make_plan.sql

```diff
@@ -0,0 +1,129 @@
+create or replace function make_plan(path text) 
+    returns table
+            (
+                migration_filename  text,
+                migration_statement omni_sql.statement
+            )
+    language plpgsql
+as
+$$
+declare
+    local_conn_info text := 'host=localhost port=' || (select setting from pg_settings where name = 'port') || ' user=' || current_user;
```

Comment:
In other parts (omni_test) I've used unix port instead because I suspect it is even more commonly available in local setups. Probably not a big deal, but just as a note:

https://github.com/omnigres/omnigres/blob/master/extensions/omni_test/src/instantiate.sql#L66-L68